gnucash/gnome/gnc-plugin-page-owner-tree.c

```diff
@@ -932,7 +932,11 @@ static int build_owner_report (GncOwner *owner, Account *acc)
         args = scm_cons (SCM_BOOL_F, args);
     }
 
-    arg = SWIG_NewPointerObj(owner, SWIG_TypeQuery("_p__gncOwner"), 0);
+    swig_type_info * qtype = SWIG_TypeQuery("_p__gncOwner");
+    g_return_val_if_fail (qtype, -1);
+
+    arg = SWIG_NewPointerObj(owner, qtype, 0);
+
```

Comment:
`_p__gncOwner` is a `#define` in swig-engine.cpp (a generated file) so the only way that `SWIG_TypeQuery` could fail would be a link failure at compile time.

Reply:
> SWIG_TypeQuery() is looking for the type at runtime, not compile time.
You misunderstood: While SWIG_TypeQuery() is indeed a runtime check, it's checking a compile-time constant so as long as the correct swig library is linked in *it will always succeed* and if the correct swig library isn't linked in *the build will fail*. 

> I would prefer to keep it consistent with the code above, which does a similar type query for _p_Account.

If there's a check on `_p__Account` then remove it as unnecessary. That will still satisfy your preference for consistency.